docs/connections/connector-types/s3-connector.md

```diff
@@ -0,0 +1,15 @@
+---
+title: S3 Connector
+---
+
+Macrometa collection connectors allow you to send data to an existing AWS S3 collection.
+
+## Prerequisites
+
+- Macrometa account with admin permissions.
+- Remotely accessible, running AWS S3 instance.
+- Source collections must not be empty.
```

Comment:
@praminda-macrometa - Don't we need a S3 account as a prerequisites?

Reply:
Updated.